firrtl/src/main/scala/firrtl/ir/Serializer.scala

```diff
@@ -292,6 +292,14 @@ object Serializer {
       s(init); s(info)
     case DefInstance(info, name, module, _) =>
       b ++= "inst "; b ++= legalize(name); b ++= " of "; b ++= legalize(module); s(info)
+    case DefInstanceChoice(info, name, default, option, choices) =>
+      b ++= "inst "; b ++= legalize(name); b ++= " of "; b ++= legalize(default);
+      b ++= " "; b ++= legalize(option); s(info)
+      choices.foreach {
+        case (choice, module) =>
+          newLineAndIndent(1)
+          b ++= legalize(choice); b ++= " => "; b ++= legalize(module)
+      }
```

Comment:
Is this the right textual format? `s/inst/inst_choice/`, nittily comma after the default, missing colon after option name.